pkg/proxy/util/conntrack.go

```diff
@@ -31,17 +31,21 @@ const noConnectionToDelete = "0 flow entries have been deleted"
 
 // DeleteServiceConnection uses the conntrack tool to delete the conntrack entries
 // for the UDP connections specified by the given service IPs
-func DeleteServiceConnections(execer exec.Interface, svcIPs []string) {
+func DeleteServiceConnections(execer exec.Interface, svcIPs []string) error {
+	var errs []error
 	for _, ip := range svcIPs {
-		glog.V(2).Infof("Deleting connection tracking state for service IP %s", ip)
 		err := ExecConntrackTool(execer, "-D", "--orig-dst", ip, "-p", "udp")
 		if err != nil && !strings.Contains(err.Error(), noConnectionToDelete) {
 			// TODO: Better handling for deletion failure. When failure occur, stale udp connection may not get flushed.
 			// These stale udp connection will keep black hole traffic. Making this a best effort operation for now, since it
 			// is expensive to baby-sit all udp connections to kubernetes services.
-			glog.Errorf("conntrack returned error: %v", err)
+			errs = append(errs, fmt.Errorf("Failed to delete connection tracking state for UDP service IP: %s, error: %v", ip, err))
 		}
 	}
+	if len(errs) != 0 {
+		return utilerrors.NewAggregate(errs)
```

Comment:
```
func DeleteServiceConnections(execer exec.Interface, svcIPs []string) error
```

The input of this function is IP slice, so this function will delete multiple connection conntracks. Current implementaion doesn't return immediately when an error occur. So, we probably will have more than on errors from this function.

Is there any problem?